### ISSUE TITLE
Collect canvas ads with screengrab

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#832]

**Comment by mneunomne** [comment]
@dhowe I was checking this the other day, making some tests. The tricky thing is that the canvas needs to be visible for it to be grabbed, our cosmetic filters dont enable that. Plus to grab image from canvas is super processing heavy.

**Comment by mneunomne** [comment]
I don't think this would be worth the efford + it would be too heavy to run locally. Closing.
